addons/website_sale/static/src/js/website_sale.js

```diff
@@ -684,6 +684,33 @@ publicWidget.registry.WebsiteSaleLayout = publicWidget.Widget.extend({
     },
 });
 
+publicWidget.registry.WebsiteSaleAccordionProduct = publicWidget.Widget.extend({
+    selector: "#product_accordion",
+
+    /**
+     * @override
+     */
+    async start() {
+        await this._super(...arguments);
+        this._updateAccordionActiveItem();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _updateAccordionActiveItem: function () {
+        const firstAccordionItemEl = this.$target[0].querySelector(".accordion-item");
+        const firstAccordionItemButtonEl = firstAccordionItemEl.querySelector(".accordion-button");
+        firstAccordionItemButtonEl.classList.remove("collapsed");
```

Comment:
Nitpicking: add and empty line after variables definition. Not blocking, can be addressed/discussed during tech.review 

```suggestion
        const firstAccordionItemButtonEl = firstAccordionItemEl.querySelector(".accordion-button");
        
        firstAccordionItemButtonEl.classList.remove("collapsed");
```